src/Spago/Watch.hs

```diff
@@ -18,14 +18,15 @@ import qualified System.FSNotify        as Watch
 import           System.IO              (getLine)
 import qualified UnliftIO
 import           UnliftIO.Async         (race_)
+import           Data.Time.Clock        (NominalDiffTime, getCurrentTime, diffUTCTime)
 
 -- Should we clear the screen on rebuild?
 data ClearScreen = DoClear | NoClear
   deriving Eq
 
 watch :: Spago m => Set.Set Glob.Pattern -> ClearScreen -> m () -> m ()
 watch globs shouldClear action = do
-  let config = Watch.defaultConfig { Watch.confDebounce = Watch.Debounce 0.1 } -- in seconds
+  let config = Watch.defaultConfig
```

Comment:
If we're skipping the `fsnotify` debouncing should we use `Watch.NoDebounce` in the config? Or are we adding our own debouncing on top of that?